### DISCUSSION TITLE
For you (beta): Recommended based on people you follow

### DISCUSSION THREAD

**Discussion by dilshankarunarathne** [community/community#13337] [Other Feature Feedback, Questions, & Ideas] [answered]
I usually follow people when they appear on https://github.com/. I look at their profile picture, bio and most importantly - how many others follow this person.
But this new Beta, do not show followers - following information. Without that, I feel reluctant to follow someone. 
Please bring that feature back, I do not like this new style.

**Comment by jfullstackdev** [comment, marked as answer]
not very sure of your request but

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
